deploy/scripts/src/cri-lime/test.ts

```diff
@@ -46,7 +46,39 @@ const profiles: ProfileList = {
       exec: 'passport'
     }
   },
-  lowVolumeTest: {
+  bau1x: {
+    passport: {
+      executor: 'ramping-arrival-rate',
+      startRate: 1,
+      timeUnit: '1m',
+      preAllocatedVUs: 1,
+      maxVUs: 8,
+      stages: [
+        { target: 15, duration: '5m' }, // Ramp up to 15 iterations per minute in 5 minutes
```

Comment:
This is way above production volumes.  I think we're seeing ~136 users per hour, as a maximum load on the passport CRI.
This scenario has 900 people in the first 5 minutes...